### DISCUSSION TITLE
Does fudgeo support raster layers read/insertion ?

### DISCUSSION THREAD

**Discussion by nmoreaud** [realiii/fudgeo#71] [Q&A] [answered]
Does fudgeo support raster layers read/insertion ?
I'm looking for a library that can fast insert raster layers into a geopackage.

**Comment by jlhumber** [comment, marked as answer]
At the moment `fudgeo` does not have any raster support, it is on the wish list but no timeline associated with implementation.   In issue #66  there is a link to another repository that can supposedly handle rasters, we have not tried or verified that code.
